vm/actor/tests/common/mod.rs

```diff
@@ -80,6 +81,12 @@ impl<'a, BS: BlockStore> MockRuntime<'a, BS> {
             actor_code_cids: HashMap::new(),
             new_actor_addr: None,
 
+            message: UnsignedMessage::builder()
+                .to(Address::new_id(100))
+                .from(Address::new_id(100))
```

Comment:
So the issue still remains that there is duplicate data (from and to address) which will cause issues with certain tests. If you intend to keep that `message()` function implementation to returning this message, if somewhere in the code they retrieved this to and from address it would be wrong. The equivalents of `to` and `from` here are `receiver` and `caller` in the mockrt respectively